lib/elixir/lib/supervisor.ex

```diff
@@ -546,6 +564,13 @@ defmodule Supervisor do
 
     * a two-element tuple in the shape of `{module, arg}`, where `module.child_spec(arg)`
       will be invoked to retrieve its child specification
+  """
+  @type children :: [child_spec() | module | {module, arg :: term}]
```

Comment:
It is best to call it `child` and then pass `[child]`. This is because if somewhere we expect this as a single argument, we won't need to create a new type.

Reply:
I thought of this, but child is taken

Reply:
Maybe name it `child_element`?

Reply:
So calling this `children` is not helpful either, as both will be confusing.

I actually recall now that I chose explicitly about not introducing `children()`, because we wanted to list ` child_spec() | {module, term} | module` on the signatures for clarity...

Reply:
`child_reference` ?

Reply:
Sometimes a tiny bit of verbosity can be easier for readers than a tiny bit more of indirection 😉